crates/core/tedge_watchdog/src/systemd_watchdog.rs

```diff
@@ -118,15 +119,21 @@ async fn start_watchdog_for_tedge_services(tedge_config_dir: PathBuf) {
         let service_name = service.default_service_name().unwrap();
         match get_watchdog_sec(&format!("/lib/systemd/system/{service_name}.service")) {
             Ok(interval) => {
-                let req_topic = format!("tedge/health-check/{service_name}");
+                let req_topic = mqtt_schema.topic_for(
+                    &service,
+                    &Channel::Command {
+                        operation: OperationType::Health,
+                        cmd_id: "check".to_string(),
```

Comment:
In one of the older issues, we had started a discussion to reconsider this decision to use the `cmd` topics for health check as well, as it doesn't follow all the other rules associated with commands, like unnecessary retained `init` messages, clearing them later etc. We had kinda agreed on the need for another "prompting mechanism" where the request can be sent on one topic and the response can be something completely independent, without linking the request and response. Should we consider that for this PR, before this API released?

If yes, we could introduce a `te/.../req/health_check` topic for health checks and other `req` types can be used to request/trigger other kinds of actions in future as well.

Reply:
That's kind of a big change and very close to planned release... This needs to be discussed with @reubenmiller

Reply:
I'm in favor of this change to be done for the 1.0 release, but definitely not for this PR.